catt/controllers.py

```diff
@@ -102,25 +114,10 @@ def __init__(self, device_name):
                 raise ValueError
             self.cast = pychromecast.Chromecast(cached_ip)
         except (pychromecast.error.ChromecastConnectionError, ValueError):
-            if device_name:
-                self.cast = self.get_chromecast(friendly_name=device_name)
-            else:
-                self.cast = self.get_chromecast()
+            self.cast = get_chromecast(device_name)
             cache.set(self.cast.name, self.cast.host)
         time.sleep(0.2)
 
-    def get_chromecast(self, friendly_name=None):
```

Comment:
Ah, I meant to investigate: How did this even work, if it wasn't called on an instance? I can only guess `self` was actually the class argument there.